osu.Game/Overlays/Comments/CommentsContainer.cs

```diff
@@ -114,9 +110,10 @@ public CommentsContainer()
         }
 
         [BackgroundDependencyLoader]
-        private void load()
+        private void load(OverlayColourProvider colourProvider)
         {
-            background.Colour = colours.Gray2;
+            background.Colour = colourProvider.Background5;
```

Comment:
pretty sure you can do the same refactor here. generally drawable hierarchies should be constructed in BDL not the constructor, because it allows for better async support.

Reply:
I wasn't sure about that, will apply that in my future pr's as well